service/src/java/org/apache/hive/service/cli/thrift/ThriftBinaryCLIService.java

```diff
@@ -113,43 +113,68 @@ protected void initServer() {
       // TCP Server
       server = new TThreadPoolServer(sargs);
       server.setServerEventHandler(new TServerEventHandler() {
+
         @Override
         public ServerContext createContext(TProtocol input, TProtocol output) {
           Metrics metrics = MetricsFactory.getInstance();
           if (metrics != null) {
-            try {
```

Comment:
Do we really want to swallow exceptions in Metrics system, if the metrics should be working?  Why do we want this?
Thanks,  Peter

Reply:
@pvary Thanks for the review!

I removed this because there is another place in this same class that does not wrap the calls in a `try` block so I am trying to keep in consistent, but also, the method `incrementCounter` does not throw anything.

https://github.com/apache/hive/blob/master/common/src/java/org/apache/hadoop/hive/common/metrics/common/Metrics.java#L71